example/crawl_async_example.dart

```diff
@@ -26,9 +26,9 @@ Future<Null> main() async {
   print(allImports.map((s) => s.uri).toList());
 }
 
-AnalysisContext _analysisContext;
+AnalysisContext? _analysisContext;
 
-Future<AnalysisContext> get analysisContext async {
+Future<AnalysisContext?> get analysisContext async {
```

Comment:
This should not be nullable I don't think, we should instead add a `!` if we need to before returning values or do a non-null check and throw a StateError or something like that.

Reply:
sure